### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/Kiss/actions/runs/4861480826#summary-13172214576

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/Kiss/actions/runs/4861481162#summary-13172215275

### CODE REVIEW

File: manual/jsdoc/AGGrid.html [hunk 1]
```diff
@@ -3474,7 +3474,7 @@ <h4 class="name" id="getSelectedRowIndex"><span class="type-signature"></span>ge
 
 <div class="description usertext">
     Return the index of the selected row.  <code>null</code> is returned if no row is selected.
-If multiple rows are selected, the index of the forst selected row is returned.
+If multiple rows are selected, the index of the first selected row is returned.
```
Comment (jsoref) [inline manual/jsdoc/AGGrid.html:3477]: this typo is present in a number of locations. I didn't see any particular indication that it was intentional as opposed to copy+paste.

File: manual/jsdoc/DateUtils.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h2>
         DateUtils
         </h2>
         
-            <div class="class-description usertext">Function names are meant to tell what the input and output date date types they work on.
+            <div class="class-description usertext">Function names are meant to tell what the input and output date types they work on.
```
Comment (jsoref) [inline manual/jsdoc/DateUtils.html:51]: Sometimes duplicate words are intentional. I don't think this is such a case.

File: src/main/core/org/kissweb/JsonPath.java [hunk 3]
```diff
@@ -400,7 +400,7 @@ public static Integer getDate(JSONObject jobj, String path) {
     public static void main(String [] argv) throws IOException {
         JSONObject jobj = toJson("input.json");
         JSONObject jobj2 = getObject(jobj, "context.System.application");
-        String s1 = getString(jobj, "context.Systsem.application.applicationId");
+        String s1 = getString(jobj, "context.System.application.applicationId");
```
Comment (jsoref) [inline src/main/core/org/kissweb/JsonPath.java:403]: This is notable...

**Review by jsoref — COMMENTED** [review]
Some corrections were automatically suggested by Google Sheets. A number weren't.

For this project I added a [lisp dictionary](https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20230502/dictionaries/lisp/lisp.txt).

All fault mine.

**Comment by blakemcbride** [comment]
Thank you very much!

**Review by blakemcbride — APPROVED** [review]
Thank you!

**Pull request merged**